src/server/game/Entities/Player/Player.cpp

```diff
@@ -18672,31 +18672,31 @@ bool Player::Satisfy(AccessRequirement const* ar, uint32 target_map, bool report
 
 bool Player::CheckInstanceLoginValid(Map* map)
 {
-    if (!map->IsDungeon() || IsGameMaster())
+    if (!map->IsDungeon() || IsInstanceLoginGameMasterException())
```

Comment:
Yes, player will receive message even if he is in raid group. But we can send this warning-message only if player is not in group, if needed, just add (!GetGroup()) check in IsInstanceLoginGameMasterException